Add Faq Section for RN 2.0 and Swift (#2972)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/react-native/index.md

- [Facebook App Events](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-facebook-app-events){:target="_blank"}
- [Firebase](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-firebase){:target="_blank"}

## FAQs
### Can I use the catalog of device-mode destinations from Segment's 1.X.X React-Native release?
No, only the plugins listed above are supported in device-mode for Analytics React Native 2.0.
### Will I still see device-mode integrations listed as `false` in the integrations object?

> When you successfully package a plugin in device-mode, you will no longer see the integration listed as `false` in the integrations object for a Segment event. This logic is now packaged in the event metadata, and is not surfaced in the Segment debugger.

Reviewer: ```suggestion
When you successfully package a plugin in device-mode, you won't see the integration listed as `false` in the integrations object for a Segment event. This logic is packaged in the event metadata, and isn't surfaced in the Segment debugger.
```
